### DISCUSSION TITLE
v1.8.2 -Right/Left Split EQ, Audio Signal Visualizations, Volume Mapping

### DISCUSSION THREAD

**Discussion by nodeful** [bitgapp/eqMac#891] [General]
* Separate Right/Left channel processing with Expert EQ
* Frequency Volume visualization in Advanced and Expert EQ
* Per App volume visualization in App Mixer
* Volume Remapping - Limit Max Volume or Adjust the Boost amount
* Adjust processing Stop delay
* Improved background CPU usage
* Fixed Facetime calls microphone volume ducking
* Various other Bug Fixes and Improvements

<hr /><em>This discussion was created from the release <a href='https://github.com/bitgapp/eqMac/releases/tag/'>v1.8.2 -Right/Left Split EQ, Audio Signal Visualizations, Volume Mapping</a>.</em>

**Comment by chrisfong** [comment]
Not working for me on MacOS Sonoma with my M2 Macbook Air. No audio to my monitor or laptop speakers when routed through eqMac. Downgrading back to v1.7.10 works for me.

**Comment by ernestp** [comment]
Long delays, UI hangs, requesting install drivers after each reboot on macOS Sonoma 14.2.1. Downgrading.

**Comment by nodeful** [comment]
Hey guys, indeed there were some flaws with v1.8.2 and over the past week I have been trying to iron them out and release v1.8.3:
https://github.com/bitgapp/eqMac/releases/tag/v1.8.3
Would be good to hear any feedback, sorry for the inconvenience!

**Reply by ernestp** [reply]
Thanks for update, checked app looks better, no hanging, UI is responsive. 
However it still requires to install the driver on reboot. 

Another issue is switching between devices. It nicely switches to Bluetooth headphones when it connected.
However when turned off eqMac switching to device that is not last used, lower in priority and have auto connect turned off.
Tried clear cached devices, didn't helped.

**Reply by ernestp** [reply]
@nodeful It seems issue with need to reinstall the driver after reboot is finally solved on v1.8.7, driver v2.5.3 on Sonoma 14.2.1
